### DISCUSSION TITLE
[Feature request]: Debian 12.8 installation

### DISCUSSION THREAD

**Discussion by robin-physics** [OpenTOPAS/OpenTOPAS#105] [Feature-Request]
### Feature request

Hi everyone

Looking to setup a OpenTopas installation shortly on several workstations. 
Has anyone had issues with using the current Debian 12.8 distribution?
Or am I best to go back to Debian V11.x?

One minor issue is finding the ISO images for earlier versions of Debian within the long directory structure of the Debian mirrors.
A bit more of a search found up to V11.11. But interested to know if later versions OK?

### TOPAS category

Outcome model

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi Robin,

I personally haven't tested using Debian 12. The current OpenTOPAS version should work fine on Debian 10 and 11 but we can't make any guarantees about Debian 12 at the moment.

If it's not too much trouble, I would recommend using Debian 11 on all the workstations.

All the best,
Thongchai

**Reply by robin-physics** [reply]
Thanks for that - going to install on a few older retired desktop PCs over the next week and get OpenTOPAS up and running for a student project in the new year.
